hazelcast/reactor.py

```diff
@@ -38,6 +33,22 @@
 
 _logger = logging.getLogger(__name__)
 
+# We should retry receiving/sending the message in case of these errors
+# EAGAIN or EWOULDBLOCK: The read/write would block
```

Comment:
nit: [manpages](https://man7.org/linux/man-pages/man3/errno.3.html) say that EAGAIN in general means "Resource temporarily unavailable", but it might be (but not always) the same as EWOULDBLOCK.

Reply:
Separated the explanations for them